apps/experiments/views/experiment.py

```diff
@@ -1464,3 +1468,26 @@ def get_release_status_badge(request, team_slug: str, experiment_id: int):
     experiment = get_object_or_404(Experiment, id=experiment_id, team=request.team)
     context = {"has_changes": experiment.compare_with_latest(), "experiment": experiment}
     return render(request, "experiments/components/unreleased_badge.html", context)
+
+
+@login_required
```

Comment:
This view also requires a team:
```suggestion
@login_and_team_required
```